README.md

```diff
@@ -1 +1,157 @@
-# dagster-lightly-example
\ No newline at end of file
+# Data Pre-processing Pipeline on AWS with Dagster 
+
+Data collection and pre-processing pipelines have become increasingly automated in recent years. The Lightly solution is an intelligent system designed to process raw, unlabeled image data, select the most informative samples for labeling, and mitigate dataset bias.
```

Comment:
> The Lightly solution is an intelligent system designed to process raw, unlabeled image data, select the most informative samples for labeling, and mitigate dataset bias.

This sentence is copy pasted from the introduction of the docs. That's a bit strange. I'd just use something like
> Lightly's ability to integrate with Dagster makes it a great building block in your machine learning pipeline.

Or something like that.

Reply:
*for your